src/Security/Security.php

```diff
@@ -948,7 +948,7 @@ protected function renderWrappedController($title, array $fragments, array $temp
         static::clearSessionMessage();
 
         // Ensure title is present - in case getResponseController() didn't return a page controller
-        $fragments = array_merge($fragments, ['Title' => $title]);
+        $fragments = array_merge(['Title' => $title], $fragments);
```

Comment:
```suggestion
        $fragments['Title'] = $title;
```
Probably makes more sense to just set that in the array directly.

Reply:
The merge allows an existing title key in the fragment array to take precedence, so it needs to stay put.

Reply:
Ahh, for some reason I had that backwards. Fair enough.